File: C Programming/I TECH CLASS/for loop/factorial.c
#include <stdio.h>

int main() {
 
 int i,num,factorial;
 printf("Enter The Number Whose Factorial You Want\n");
 scanf("%d",&num);

    for(factorial=1,i=1;i<=num;i++)
    {
        factorial=factorial*i;
    }   
    printf("Factorial of %d is %d\n",num,factorial);
 
}

File: Java/whileloop/factorial.java
package Java.whileloop;

import java.util.Scanner;

public class factorial {
    public static void main(String[] args) {
        int num,factorial=1,i=1;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter The Number Whose Factorial You Want ");
        num=sc.nextInt();
        while(i<=num)
        {
            factorial=factorial*i;
            i++;
        }
        System.out.println("Factorial of "+num+" = "+factorial);
        sc.close();
    }
}
